python/ray/rllib/evaluation/tf_policy_graph.py

```diff
@@ -410,8 +410,9 @@ def _build_learn_on_batch(self, builder, postprocessed_batch):
         fetches = builder.add_fetches([
             self._apply_op,
             self._get_grad_and_stats_fetches(),
+            self.model.beholder_tensors
         ])
-        return fetches[1]
+        return fetches[1], fetches[2]
```

Comment:
Override this method in WGymVTracePolicyGraph, and remove these changes from ray.